### ISSUE TITLE
get_posts() not working for Pages

### ISSUE THREAD

**Issue by KatkaBielcikova** [kevinzg/facebook-scraper#1061]
it is working for groups
get_posts(group='thenintendoswitch') -> WORKING

but not for posts
get_posts('NintendoSwitchEurope') -> NOT WORKING

**Comment by vhoma** [comment]
Same issue. Just getting empty output with no error or warning.

**Comment by xu3cylee** [comment]
mine is working just fine though. Have you tried running with credentials or cookie.txt?

**Comment by KatkaBielcikova** [comment]
Yeah with credentials it works, but with credentials you get your account blocked after getting like 20posts. 

I think the problem with this is that the first page is "Log into Facebook" and you can decline logging in. I think this scraper does not deal with that page and expects to get the Posts page.
